golang/cmd/kafka-to-postgresql/pxCount.go

```diff
@@ -41,7 +41,7 @@ func (c Count) ProcessMessages(msg internal.ParsedMessage) (putback bool, err er
 		zap.S().Warnf("Failed to unmarshal message: %s", err.Error())
 		return false, err
 	}
-	if !internal.IsValidStruct(sC, []string{}) {
+	if !internal.IsValidStruct(sC, []string{"Scrap"}) {
```

Comment:
`scrap` or `Scrap`

Reply:
Scrap, this is referencing the field name, not the json name, see pxAddShift:
https://github.com/united-manufacturing-hub/united-manufacturing-hub/blob/060efd50c2821e8b44de77c4fabeb832fda93449/golang/cmd/kafka-to-postgresql/pxAddShift.go#L42-L45